etc/docs/generate-docs.ts

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env ts-node
+
+import { parse, stringify } from '@iarna/toml';
+import * as child_process from 'child_process';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import { promisify } from 'util';
+
+const exec = promisify(child_process.exec);
+
+const RELEASES_TOML_FILE = './template/data/releases.toml';
+const RELEASES_JSON_FILE = './template/static/versions.json';
+
+interface JsonVersionSchema {
+  version: string;
+}
+
+interface VersionSchema {
+  version: string;
+  status: 'current' | 'supported' | 'not-supported';
+  api: string;
+  usesMongoDBManual?: boolean;
+  docs?: string;
+  semvarVersion: string;
+}
+
+interface TomlVersionSchema {
+  current: string;
+  mongodDBManual: string;
+  versions: VersionSchema[]
+}
+
+/**
+ * ATTENTION - EDIT THIS BEFORE RELEASING
+ */
+const NEW_VERSION: VersionSchema = {
+  // version - the display name for the version of the driver on the docs website
+  //  ex: Driver 4.5
+  version: 'Driver 4.5',
+
+  // status - a status for the version.  typically would probably just be 'latest'
+  status: 'current',
+
+  // api - the name of the folder for the generated documentation to live.  typically should be version_id prefixed with `./`
+  //  ex: ./4.5
+  api: './4.5',
+
+  // usesMongoDBManual - if true, includes a link to the mongodb documentation for the new version
+  usesMongoDBManual: true,
```

Comment:
I noticed this always links to latest docs instead of that version's specific docs, is that intentional? might be a question for @nbbeeken

Reply:
I'm not sure.  My schema in the script simply matches the existing toml data used to generate our static site.  If looking into this is something we want to include in this work, I can investigate

Reply:
I was hoping Neal had a quick answer there

Reply:
It's always linked to the latest, we could add versioning but not here I think.